src/com/webonise/polygonassignment/Polygon.java

```diff
@@ -0,0 +1,17 @@
+package com.webonise.polygonassignment;
+
+public abstract class Polygon {
+    protected int noOfSides;
+    protected double lengthOfSides;
+
+    Polygon(int noOfSides) {
+        this.noOfSides = noOfSides;
+    }
+
+    Polygon(int noOfSides, double lengthOfSides) {
+        this.noOfSides = noOfSides;
+        this.lengthOfSides = lengthOfSides;
+    }
+
+    abstract void CalculateResultAndPrint();
```

Comment:
Please follow naming conventions method name should be `calculateResultAndPrint`